### PR TITLE
OSDOCS-1050 Power and Z Samples release note

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-1050

### CODE REVIEW

**Comment by bmcelvee** [comment]
Preview build: https://osdocs-1050-samples-power-z-rn--ocpdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes.html#ocp-4-6-samples-operator

**Comment by bmcelvee** [comment]
@adambkaplan would you mind reviewing this release note, please? Thanks!

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -482,6 +482,11 @@ and no other Reports or ReportQueries depend on the expiring Report, the Meterin
 removes the Report from your cluster at the end of its retention period. For more information,
 see metering Reports xref:../metering/reports/metering-about-reports.adoc#metering-expiration_metering-about-reports[expiration].
 
+[id="ocp-4-6-samples-operator"]
+==== Samples Operator on Power and Z
+
+The Samples Operator is now installed on Power and Z, and delivers Power and Z samples including images, imagestreams, and templates.
```
Comment (adambkaplan) [inline release_notes/ocp-4-6-release-notes.adoc:488]: Technically the Samples Operator exists on Power and Z, it just bootstraps as `Removed`. 
```suggestion
Imagestreams and templates for Power and Z architectures are now available and installed by the Samples Operator by default..
```

**Review by adambkaplan — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (bmcelvee) [reply release_notes/ocp-4-6-release-notes.adoc:488]: Thank you! Updating now.

**Review by sfortner-RH — APPROVED** [review]
LGTM!

**Pull request merged**